### PR TITLE
60 update exportinputdialog component

### LINKED ISSUE #60: Update ExportInputDialog component
### Tasks
- [x] Folderize
- [ ] Create unit tests
- [x] Convert to CSS Modules
- [x] Convert to functional component
- [x] Add PropTypes and Default Props
- [x] Convert to Typescript

### PR DESCRIPTION
**Pull request by dpenciso**

### CODE REVIEW

File: src/components/ExportInputDialog/ExportInputDialog.tsx [hunk 1]
```diff
@@ -1,110 +1,95 @@
...
-  button: {
-    margin: theme.spacing(1),
-  },
-}));
+//   return (
+//     <div
+//       role="tabpanel"
+//       hidden={value !== index}
+//       id={`full-width-tabpanel-${index}`}
+//       aria-labelledby={`full-width-tab-${index}`}
```
Comment (Incin) [inline src/components/ExportInputDialog/ExportInputDialog.tsx:26]: can this commented out code just be removed?

File: src/components/ExportInputDialog/ExportInputDialog.tsx [hunk 2]
```diff
@@ -1,110 +1,95 @@
...
-  children: PropTypes.node.isRequired,
-  index: PropTypes.any.isRequired,
-  value: PropTypes.any.isRequired,
-};
-
-export default function ExportInputDialog({ open, handleClose, displayInfo }) {
-  const classes = useStyles();
+interface LooseObject {
+  // TODO figure out what the exact typing is
+  [key: string]: any;
```
Comment (Incin) [inline src/components/ExportInputDialog/ExportInputDialog.tsx:41]: we should be able to figure out what this type is, i dont think we want an any here

File: src/index.js [hunk 3]
```diff
@@ -227,7 +227,8 @@ window.trelliscopeApp = trelliscopeApp;
 // trelliscopeApp('02a6e2cc', '_test/univar/config.jsonp', { logger: true });
 // trelliscopeApp('001a3be8', '_test/foundationtest/config.jsonp', { logger: true });
 // trelliscopeApp('fcf74975', '_test/gapminder_autocogs/config.jsonp', { logger: true });
-trelliscopeApp('80222985', '_test/gapminder_coggroups/config.jsonp', { logger: true });
+// trelliscopeApp('80222985', '_test/gapminder_coggroups/config.jsonp', { logger: true });
+trelliscopeApp('62e90658', '_test/gapminder_bells/config.jsonp', { logger: true });
```
Comment (Incin) [inline src/index.js]: i think we decided to not commit these changes. So lets leave it as the standard config.jsonp

**Review by Incin — CHANGES_REQUESTED** [review]

File: src/components/ExportInputDialog/ExportInputDialog.tsx (hunk 1 above)
Comment (dpenciso) [reply src/components/ExportInputDialog/ExportInputDialog.tsx:26]: jeff said to leave it commented

File: src/components/ExportInputDialog/ExportInputDialog.tsx (hunk 2 above)
Comment (dpenciso) [reply src/components/ExportInputDialog/ExportInputDialog.tsx:41]: walked through it with jeff. he said to do this

File: src/index.js (hunk 3 above)
Comment (dpenciso) [reply src/index.js]: good call

**Review by Incin — APPROVED** [review]

**Pull request merged**